README.md

```diff
@@ -1,102 +1,262 @@
-# SAML2 Library
+# SAML2-js
 
-## Description
+[![Build Status](https://ci.ops.clever.com/api/badge/github.com/Clever/saml2/status.svg?branch=master)](https://ci.ops.clever.com/github.com/Clever/saml2)
 
-Takes care of the complexities of the SAML protocol and provides an easy interface for using it. Specifically, creating metadata.xml files, creating `AuthnRequest`s and parsing and validating `AuthnResponse`s.
-
-This is exposed as both a series of functions that implement each step of the SAML protocol, and an Express middleware that creates the necessary endpoints for the metadata, the login and the assertion.
+`saml2-js` is a module that abstracts away the complexities of the SAML protocol and provides an easy interface for using it in node.
 
 ## Installation
 
 ```bash
-  npm install saml2-js
+  npm install saml2-js --save
 ```
 
-## Expected Usage
+## Usage
```

Comment:
_nit_: We have a lot of these `##` headings - thoughts on including the Installation step under usage? It's part of being able to use it.